fakelibusb_test.go

```diff
@@ -87,6 +87,8 @@ type fakeLibusb struct {
 	mu sync.Mutex
 	// fakeDevices has a map of devices and their descriptors.
 	fakeDevices map[*libusbDevice]*fakeDevice
+	// fakeLibUsbDevices keeps the order of devices to be accessd by wrapSysDevice
+	fakeLibUsbDevices []*libusbDevice
```

Comment:
Sure, good point. The FDs I randomly saw were e.g. never 0,1,2 as those are 'special'. Certainly a '0' could be messing up somewhere in a test.

Reply:
Hm, correction about the 0 - it makes no difference since the FDs still need to be iterated from the list of fakeDevices, unless we'd add a 'fakeFD' entry in there?